src/reviews/reviews.module.ts

```diff
@@ -28,11 +28,11 @@ export class ReviewsModule implements NestModule {
     consumer
       .apply(AuthenticationMiddleware)
       .forRoutes(
-        {method: RequestMethod.GET, path: '/reviews'},
-        {method: RequestMethod.GET, path: '/reviews/:id'},
-        {method: RequestMethod.POST, path: 'reviews'},
-        {method: RequestMethod.PATCH, path: '/reviews/:id'},
-        {method: RequestMethod.DELETE, path: '/reviews/:id'},
+        { method: RequestMethod.GET, path: '/reviews' },
+        { method: RequestMethod.GET, path: '/reviews/:id' },
+        { method: RequestMethod.POST, path: 'reviews' },
+        { method: RequestMethod.PATCH, path: '/reviews/:id' },
+        { method: RequestMethod.DELETE, path: '/reviews/:id' },
```

Comment:
A lot of repetitive string `reviews` here. 

Maybe we should add base route names to a constant file for reusability.